### PR TITLE
r*: normalize docs

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY

Normalize doc blocks.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

plugins/modules/read_csv.py
plugins/modules/redfish_command.py
plugins/modules/redfish_config.py
plugins/modules/redfish_info.py
plugins/modules/redhat_subscription.py
plugins/modules/redis.py
plugins/modules/redis_data.py
plugins/modules/redis_data_incr.py
plugins/modules/redis_data_info.py
plugins/modules/redis_info.py
plugins/modules/rhevm.py
plugins/modules/rhsm_release.py
plugins/modules/rhsm_repository.py
plugins/modules/riak.py
plugins/modules/rocketchat.py
plugins/modules/rollbar_deployment.py
plugins/modules/rpm_ostree_pkg.py
plugins/modules/rundeck_acl_policy.py
plugins/modules/rundeck_job_executions_info.py
plugins/modules/rundeck_job_run.py
plugins/modules/rundeck_project.py
plugins/modules/runit.py

### CODE REVIEW

**Review by jsumners — APPROVED** [review]

File: plugins/modules/redis.py [hunk 1]
```diff
@@ -8,91 +8,85 @@
...
+      - The mode of the redis instance [replica command].
+      - V(slave) is an alias for V(replica).
+    default: replica
+    choices: [master, replica, slave]
+    type: str
+    aliases:
+      - slave_mode
+  db:
+    description:
+      - The database to flush (used in db mode) [flush command].
```
Comment (felixfontein) [inline plugins/modules/redis.py:61]:
```suggestion
      - The database to flush (used in DB mode) [flush command].
```

File: plugins/modules/redis.py [hunk 2]
```diff
@@ -8,91 +8,85 @@
...
+    type: int
+  flush_mode:
+    description:
+      - Type of flush (all the dbs in a redis instance or a specific one) [flush command].
```
Comment (felixfontein) [inline plugins/modules/redis.py:65]:
```suggestion
      - Type of flush (all the DBs in a redis instance or a specific one) [flush command].
```

File: plugins/modules/redis.py [hunk 3]
```diff
@@ -8,91 +8,85 @@
...
 
 notes:
-   - Requires the redis-py Python package on the remote host. You can
-     install it with pip (pip install redis) or with a package manager.
-     https://github.com/andymccurdy/redis-py
-   - If the redis master instance we are making replica of is password protected
-     this needs to be in the redis.conf in the masterauth variable
-
+  - Requires the C(redis-py) Python package on the remote host. You can install it with pip
+    (pip install redis) or with a package manager. U(https://github.com/andymccurdy/redis-py).
```
Comment (felixfontein) [inline plugins/modules/redis.py:81]:
```suggestion
    (C(pip install redis)) or with a package manager. U(https://github.com/andymccurdy/redis-py).
```

File: plugins/modules/rhevm.py [hunk 4]
```diff
@@ -8,151 +8,150 @@
...
+    type: int
+    default: 1
+  vm_ha:
+    description:
+      - To make your VM High Available.
+    type: bool
+    default: true
+  disks:
+    description:
+      - This option uses complex arguments and is a list of disks with the options name, size and domain.
```
Comment (felixfontein) [inline plugins/modules/rhevm.py:121]:
```suggestion
      - This option uses complex arguments and is a list of disks with the options O(name), O(size), and O(domain).
```

File: plugins/modules/rhevm.py [hunk 5]
```diff
@@ -8,151 +8,150 @@
...
+    type: list
+    elements: str
+  ifaces:
+    description:
+      - This option uses complex arguments and is a list of interfaces with the options name and vlan.
```
Comment (felixfontein) [inline plugins/modules/rhevm.py:126]:
```suggestion
      - This option uses complex arguments and is a list of interfaces with the options O(name) and O(vlan).
```

File: plugins/modules/rhevm.py [hunk 6]
```diff
@@ -8,151 +8,150 @@
...
+    type: int
+    default: 1
+  vm_ha:
+    description:
+      - To make your VM High Available.
+    type: bool
+    default: true
+  disks:
+    description:
+      - This option uses complex arguments and is a list of disks with the options O(name), O(size), and O(domain).
```
Comment (russoz) [inline plugins/modules/rhevm.py:121]: I reckon these "options" mentioned are actually non-declared suboptions. Changing the markup to `V()`.
```suggestion
      - This option uses complex arguments and is a list of disks with the options V(name), V(size), and V(domain).
```

File: plugins/modules/rhevm.py [hunk 7]
```diff
@@ -8,151 +8,150 @@
...
+    type: list
+    elements: str
+  ifaces:
+    description:
+      - This option uses complex arguments and is a list of interfaces with the options O(name) and O(vlan).
```
Comment (russoz) [inline plugins/modules/rhevm.py:126]:
```suggestion
      - This option uses complex arguments and is a list of interfaces with the options V(name) and V(vlan).
```

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks!

**Pull request merged**